specification/validated-relationships-service-api.yaml

```diff
@@ -199,6 +230,236 @@ paths:
                 $ref: '#/components/schemas/OperationOutcome'
 components:
   schemas:
+    QuestionnaireResponse:
+      type: object
+      description: A FHIR QuestionnaireResponse
+      properties:
+        resourceType:
+          type: string
+          description: The FHIR resource type
+          enum: ["QuestionnaireResponse"]
+        status:
+          type: string
+          description: The position of the questionnaire response within its overall lifecycle.
+          enum: ["completed"]
+        authored: 
+          type: string
+          description: The date and/or time that this set of answers were last changed.
+          # On Confluence the example has this as a string but here:
+          # https://simplifier.net/packages/hl7.fhir.r4.core/4.0.1/files/81730/~details
+          # it looks like an object
+        source: 
+          type: object
+          description: The person who answered the questions about the subject.
+          properties:
+            type:
+              type: string
+              enum: [RelatedPerson] # Here: https://simplifier.net/packages/hl7.fhir.r4.core/4.0.1/files/81189/~details this looks like it should be a URI
```

Comment:
Don't understand what does this doing, is this correct?

Reply:
It's not clear to me either. In the example on confluence it is just `RelatedPerson` as you say but the spec seems to be expecting a URI to a specific `RelatedPerson`